tests/golden/check-local-refs/expected2.gold

```diff
@@ -139,7 +139,7 @@
        - link: /b/../../b.md
        - anchor: -
 
-     ⛀  Link targets a local file outside repository:
+     Link targets a local file outside repository:
         dir1/b/../../b.md
```

Comment:
Seems like we ended up with 3-spaces indentation at lines like this one.

Probably this made sense before because `⛀` was a wide character, but now this catches an eye. What do you think?

Reply:
Right. I didn't notice it. I can replace all these 3-space usages by 2-spaces.

Reply:
Nice, please do this replacement.